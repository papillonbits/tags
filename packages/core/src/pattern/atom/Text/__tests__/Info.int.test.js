import { regular } from './Info.int.story'

describe('<Info />', () => {
  describe('Render', () => {
    test('must match regular', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
