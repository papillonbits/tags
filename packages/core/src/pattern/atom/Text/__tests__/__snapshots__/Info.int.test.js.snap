// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Info /> Render must match regular 1`] = `
<div>
  <span
    class="info"
  >
    You will receive a notification with every payment of your credits
  </span>
</div>
`;
